File: benchmarks/array-100K/Main.java
public class Main {

    public static void main(String[] args) {
        long before = System.nanoTime();

        int[] arr = new int[100_000];
        for (int i = 0; i < 100_000; i++) {
            arr[i] = i;
        }

        long after = System.nanoTime();

        System.out.println(after - before);
    }

}

File: benchmarks/array-100K/main.cpp
#include <iostream>
#include <chrono>

int main() {
    const auto before = std::chrono::steady_clock::now();

    int arr[100000];
    for (int i = 0; i < 100000; i++) {
        arr[i] = i;
    }

    const auto after = std::chrono::steady_clock::now();
    const auto duration = after - before;

    std::cout << std::chrono::duration_cast<std::chrono::nanoseconds>(duration).count() << std::endl;

    return 0;
}